wp-modules/app/app.php

```diff
@@ -25,10 +28,14 @@ function get_app_state() {
 		'patternCategories' => \WP_Block_Pattern_Categories_Registry::get_instance()->get_all_registered(),
 		'apiNonce'          => wp_create_nonce( 'wp_rest' ),
 		'apiEndpoints'      => array(
-			'deletePatternEndpoint' => get_rest_url( false, 'pattern-manager/v1/delete-pattern/' ),
+			'deletePatternEndpoint'         => get_rest_url( false, 'pattern-manager/v1/delete-pattern/' ),
+			'updateDismissedThemesEndpoint' => get_rest_url( false, 'pattern-manager/v1/update-dismissed-themes/' ),
 		),
 		'siteUrl'           => get_bloginfo( 'url' ),
 		'adminUrl'          => admin_url(),
+		'themeName'         => wp_get_theme()->get( 'Name' ),
+		'versionControl'    => check_for_version_control_in_theme(),
+		'dismissedThemes'   => get_dismissed_themes(),
```

Comment:
How about:

```diff
diff --git a/wp-modules/app/app.php b/wp-modules/app/app.php
index b064a229..46abb394 100644
--- a/wp-modules/app/app.php
+++ b/wp-modules/app/app.php
@@ -33,9 +33,7 @@ function get_app_state() {
                ),
                'siteUrl'           => get_bloginfo( 'url' ),
                'adminUrl'          => admin_url(),
-               'themeName'         => wp_get_theme()->get( 'Name' ),
-               'versionControl'    => check_for_version_control_in_theme(),
-               'dismissedThemes'   => get_dismissed_themes(),
+               'showVersionControlNotice' => check_for_version_control_in_theme(),
        );
 }

``` 

(With my diff, this will also need `npm run lint:php:fix`)

Reply:
The front-end doesn't need to know all of the themes that have been dismissed. It just needs to know whether to show the version control notice.

Reply:
This was great feedback! [37afb1a](https://github.com/studiopress/pattern-manager/pull/145/commits/37afb1a1d1a8d1d7419462840ca3fdb263ef7b38) implements this.